pkg/launcher/paths.go

```diff
@@ -103,12 +103,7 @@ func DetermineRootDirectoryOverride(slogger *slog.Logger, optsRootDirectory, kol
 	// unlikely path but doesn't feel right updating the rootDirectory without knowing what's going
 	// on here
 	if err != nil {
-		slogger.Log(context.TODO(), slog.LevelError,
-			"encountered error checking for pre-existing launcher.db",
-			"location", optsDBLocation,
-			"err", err,
-		)
-
+		// we should add logs here when available
```

Comment:
Maybe throw in a link to https://github.com/kolide/launcher/issues/1698 in this comment and the one on 129? And I'll update that issue now